addons/website/static/src/scss/website.scss

```diff
@@ -1154,3 +1154,45 @@ table.table_desc tr td {
 .modal-footer > .float-left {
     margin-right: auto;
 }
+
+.dropdown-menu {
```

Comment:
I don't think any css is needed in this task for now, standard bootstrap should be enough I think